backend/app/src/main/java/com/moh/phlat/backend/controller/ProcessDataController.java

```diff
@@ -103,10 +94,10 @@ public class ProcessDataController {
 			}
 		}
 		
-		Page<ProcessData> entirePage = processDataService.getProcessDataWithMessages(controlTableId, rowStatus, page, itemsPerPage, filterProcess, sortOrders);
+		Page<ProcessData> processPageData = processDataService.getProcessDataWithMessages(controlTableId, rowStatus, page, itemsPerPage, filterProcess, sortOrders);
```

Comment:
```suggestion
		Page<ProcessData> processDataPage= processDataService.getProcessDataWithMessages(controlTableId, rowStatus, page, itemsPerPage, filterProcess, sortOrders);
```
since it is page of data not actual content/elements

Reply:
Changed.